src/types/spidLevel.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * SPID authentication level enum and types.
+ *
+ * @see http://www.agid.gov.it/agenda-digitale/infrastrutture-architetture/spid/percorso-attuazione
+ */
+
+import { SpidLevelEnum } from "../../generated/backend/SpidLevel";
```

Comment:
you can replace this import with the actual enum (the type will be equivalent to the one generated from the specs in the app importing this library):

```ts
export enum SpidLevelEnum {
  "https://www.spid.gov.it/SpidL1" = "https://www.spid.gov.it/SpidL1",

  "https://www.spid.gov.it/SpidL2" = "https://www.spid.gov.it/SpidL2",

  "https://www.spid.gov.it/SpidL3" = "https://www.spid.gov.it/SpidL3"
}
```

Reply:
fixed in c5dd8fc